Makefile

```diff
@@ -34,5 +34,5 @@ goreleaser.dry-run:
 
 
 doctor:
-	go run ./tool/doctor/main.go
+	go run ./go/tool/bertydoctor/
```

Comment:
as soon as https://github.com/berty/berty/pull/2621 is merged, you will also need to update the other Makefile